README.md

```diff
@@ -1,2 +1,84 @@
-# mystica
-**M**ulti bod**Y** **S**imula**T**or max**I**mal **C**oordin**A**tes	
+<h1 align="left">mystica</h1>
+
+<p align="left">
+   <a href="https://github.com/ami-iit/mystica/blob/master/LICENSE"><img src="https://img.shields.io/github/license/ami-iit/mystica" alt="Size" class="center"/></a>
+</p>
+
+**Multi bodY SimulaTor maxImal CoordinAtes**
+
+**mystica** is a library of classes and functions to simulate multibody systems.
+**mystica** computes the system evolution modeling the problem using a state definition that we define _maximal_.
+The _maximal_ representation consists of a set of _non-minimum_ variables $\textbf{q}$ complemented by holonomic constraint $g(\textbf{q})=0$.
```

Comment:
Most users will not have a plugin to render LaTeX, so this text will appear confusing. Probably you can stick to just use unicode characters, for example converting LaTeX to Unicode via https://www.unicodeit.net/ : 
```suggestion
The _maximal_ representation consists of a set of _non-minimum_ variables 𝐪 complemented by holonomic constraint g(𝐪) = 0.
```